.golangci.yml

```diff
@@ -23,5 +23,6 @@ run:
     linters-settings:
       govet:
         check-shadowing: true
+        enable-all: true
```

Comment:
Lot's of good checks in go-vet these days. Including fieldalignment, more on that later.